src/attack.c

```diff
@@ -1405,8 +1418,14 @@ attack_network (struct scan_globals *globals)
                gvm_hosts_count (hosts));
 
   gvm_hosts_free (hosts);
-  if (test_alive_hosts_only)
-    gvm_hosts_free (alive_hosts_list);
+  // if (test_alive_hosts_only)
+  //   gvm_hosts_free (alive_hosts_list);
 
+  printf ("free maindb\n");
```

Comment:
Shouldn't be g_message? or better g_debug?

Reply:
It was only for debug reasons for my special case. I remove it completely.